src/zitadel.rs

```diff
@@ -50,13 +50,34 @@ impl Zitadel {
 	/// Import a list of new users into Zitadel
 	pub(crate) async fn import_new_users(&self, users: Vec<User>) -> Result<()> {
 		for user in users {
-			let zitadel_user =
+			let mut zitadel_user =
 				user.to_zitadel_user(&self.feature_flags, &self.zitadel_config.idp_id);
 			let sync_status = self.import_user(&zitadel_user).await;
 
 			if let Err(error) = sync_status {
 				tracing::error!("Failed to sync user `{}`: {:?}", zitadel_user.log_name(), error);
-			};
+
+				if error.to_string().contains("invalid ImportHumanUserRequest_Phone") {
```

Comment:
That's not particularly nice, the error in this case is a `Status` struct which should have more structured info than a string. Have you looked into how much that can be parsed instead?

Reply:
It's possible to go one level deeper, so I updated that. For reference, this is the error when importing user:
```
Status {
        code: InvalidArgument,
        message: "invalid ImportHumanUserRequest.Phone: embedded message failed validation | caused by: invalid ImportHumanUserRequest_Phone.Phone: value does not have prefix \"+\"",
        metadata: MetadataMap {/* ... */},
        source: None,
}
```